content-service/src/test/java/io/meeds/news/service/impl/NewsServiceImplTest.java

```diff
@@ -567,7 +568,7 @@ public void testCreateDraftArticleForExistingPage() throws Exception {
     news.setOriginalBody("body");
 
     // When, Then
-    assertThrows(IllegalAccessException.class, () -> newsService.updateNews(news, "john", false, false, "draft"));
+    assertThrows(IllegalAccessException.class, () -> newsService.updateNews(news, "john", false, false, "draft", CONTENT.name().toLowerCase()));
```

Comment:
Use also NewsUtils.NewsObjectType instead of "draft", "latest_draft" ...

Reply:
done !